packages/cli/fern-definition/validator/src/rules/compatible-ir-versions/compatible-ir-versions.ts

```diff
@@ -0,0 +1,95 @@
+import { createFdrGeneratorsSdkService } from "@fern-api/core";
+import { isVersionAhead } from "@fern-api/semver-utils";
+
+import { Rule, RuleViolation } from "../../Rule";
+
+function getMaybeBadVersionMessage(
+    generatorName: string,
+    minCliVersion: string,
+    cliVersion: string
+): RuleViolation[] | undefined {
+    if (!isVersionAhead(cliVersion, minCliVersion)) {
+        return [
+            {
+                severity: "error",
+                message: `The generator ${generatorName} requires CLI version ${minCliVersion} or later (current version: ${cliVersion}).`
```

Comment:
Thoughts on rewording this to: 

`${generatorName} requires atleast CLI version ${minCliVersion}. Run fern upgrade.`